eos-update-flatpak-repos

```diff
@@ -1580,6 +1580,9 @@ def migrate_extra_data(old_ref, new_ref):
     new_deploy = new_ref.get_deploy_dir()
     new_extra = os.path.join(new_deploy, 'files', 'extra')
 
+    if not os.path.exists(old_extra):
```

Comment:
Under what situation would this code path run without the old extra-data existing?

Reply:
(I think it's fine to be robust against it not existing but I can't see how it would happen.)

Reply:
In case the ref is just a regular one not an extra-data one

Reply:
Ah, I see, the caller now calls this regardless of whether the app is extra-data or not. Gotcha!